website/pages/en/index.js

```diff
@@ -355,6 +355,79 @@ UsersSection.propTypes = {
   language: PropTypes.string,
 };
 
+const tierTitle = new Map([
+  ["gold", "Gold Sponsors"],
+  ["silver", "Silver Sponsors"],
+  ["bronze", "Bronze Sponsors"],
+  ["backers", "Backers"],
+]);
+const tierDescription = new Map([
+  [
+    "gold",
+    "Gold sponsors are those who have donated an average of $500 over the past 12 months.",
```

Comment:
I got a bit confused by the method when I first bumped into this. Was it $500 per month over the last 12 month or $500 total over the last year? We can think of rephrasing, but how about changing the method instead?

One of the goals of this section is to encourage companies to donate. For me to become a gold sponsor with the current method, I need to transfer $500×12 right on the spot. That’s quite a lot, unless I want to wait. Waiting is discouraging.

If we just look at the last month, getting into the list of sponsors is easier. I can pay $500 and see my company in the list right away. If I stop donating, my company will disappear from the list within a month, so I’d rather keep the donations running.

If there are companies donating big sums of money per year, maybe we can still calculate annual totals as a fallback method, but there should be a way of jumping into the list by paying $500 today. WDYT?